### ISSUE TITLE
Crawler every minute - Error 1020

### ISSUE THREAD

**Issue by lbenicio** [codemanki/cloudscraper#276]
_Please attempt to answer the following questions before submitting a new issue:_

  * What version of Cloudscraper are you using?
    - `node -p 'require("cloudscraper/package.json").version'`: 4.3.0
  * What version of Node.js are you using? (Please share the process information)
    - `node -p process`: v8.10.0
  * When did the problem start occurring? Few weeks
  * How often does the problem occur? Every request
  * What is the URL? https://www.tibia.com
  * Are there any similar issues? (Please share the links)
Here are the issues and solutions I tried: [268](https://github.com/codemanki/cloudscraper/issues/268), [261](https://github.com/codemanki/cloudscraper/issues/261), [266](https://github.com/codemanki/cloudscraper/issues/266), [222](https://github.com/codemanki/cloudscraper/issues/222)

- [ x] I have read the [README](https://github.com/codemanki/cloudscraper#readme). (Code [examples](https://github.com/codemanki/cloudscraper/tree/master/docs/examples#readme))

_Please share a minimal working code snippet that reproduces the problem._
<details><summary>Code snippet</summary>

```js
const tr = require('tor-request');
const Cloudscraper = require('cloudscraper').defaults({ requester: tr.request });

let url = "https://www.tibia.com";

Cloudscraper.get(url, function(html_string) {
        console.log(html_string)
});
```
</details>

<details><summary>Output</summary>

```bash
{ CloudflareError: 1020, Access Denied (Custom Firewall Rules)
    at validateResponse (/opt/tests/node_modules/cloudscraper/index.js:267:11)
    at onCloudflareResponse (/opt/tests/node_modules/cloudscraper/index.js:222:5)
    at onRequestResponse (/opt/tests/node_modules/cloudscraper/index.js:205:5)
    at Request.<anonymous> (/opt/tests/node_modules/cloudscraper/index.js:149:7)
    at Object.onceWrapper (events.js:317:30)
    at emitTwo (events.js:126:13)
    at Request.emit (events.js:214:7)
    at Request.<anonymous> (/opt/tests/node_modules/request/request.js:1161:10)
    at emitOne (events.js:116:13)
    at Request.emit (events.js:211:7)
    at IncomingMessage.<anonymous> (/opt/tests/node_modules/request/request.js:1083:12)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:111:20)
    at IncomingMessage.emit (events.js:208:7)
    at endReadableNT (_stream_readable.js:1064:12)
    at _combinedTickCallback (internal/process/next_tick.js:138:11)
    at process._tickCallback (internal/process/next_tick.js:180:9)
  name: 'CloudflareError',
  message: '1020, Access Denied (Custom Firewall Rules)' }
```
</details>

<details><summary>Background</summary>
Tried adds request trough TOR to avoid blacklisting. I'm trying to make a discord bot to help players with statistics from the site (which can be updated any minute, for example, for a character death ou level up). 
</details>

<details><summary>Help to reproduce</summary>
Make a loop with 1000 requests and your machine should get into cf firewall.

```js
const tr = require('tor-request');
const Cloudscraper = require('cloudscraper').defaults({ requester: tr.request });

let url = "https://www.tibia.com";
let max = 1000;
let error_count = 0;
let exec = 0;

function results() {
        if (exec == max) {
                console.log("Resquests failed: " + error_count + " out of " + max + ".");
                console.log(((error_count / max) * 100) + "% failure.")
        }
}

for (let i = 0; i < max; i++) {
        Cloudscraper.get(url, function(html_string, err) {
                let resp = html_string + "";
                if (resp.toLowerCase().includes("error")) {
                        error_count++;
                }
                exec++;
                results();
        });
}
```
</details>

**Comment by codemanki** [comment]
@lbenicio thanks for reporting this. Did you try these fixes https://github.com/codemanki/cloudscraper/issues/264#issuecomment-535015800 and https://github.com/codemanki/cloudscraper/issues/264#issuecomment-535018890 ?

**Comment by codemanki** [comment]
I guess this should be now fixed in `4.5.0`
